fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/AbstractPersister.java

```diff
@@ -38,32 +40,18 @@
  * @author dbernstein
  * @since 6.0.0
  */
-public abstract class AbstractPersister implements Persister {
-
-    private final Set<ResourceOperationType> resourceOperationType;
-
-    private final Set<Class<? extends ResourceOperation>> resourceOperations;
-
-    AbstractPersister(final Set<Class<? extends ResourceOperation>> resourceOperations, final Set<ResourceOperationType> resourceOperationType) {
-        this.resourceOperations = resourceOperations;
-        this.resourceOperationType = resourceOperationType;
-    }
+abstract class AbstractPersister implements Persister {
```

Comment:
This should probably be explicit with `public` unless it should be `protected`?

Reply:
It should not be visible outside the package.  `protected` does not apply to  non-inner classes